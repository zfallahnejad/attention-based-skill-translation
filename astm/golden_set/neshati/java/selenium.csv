1273080,selenium
1177636,selenium
3821981,selenium
4193730,selenium
771848,selenium
2624089,selenium
1173729,selenium
705773,selenium
108827,selenium
