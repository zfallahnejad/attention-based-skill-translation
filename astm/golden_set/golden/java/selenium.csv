1273080,53,44,0.5463917525773195
1177636,46,21,0.6865671641791045
3821981,38,46,0.4523809523809524
771848,28,32,0.4666666666666667
4193730,28,39,0.417910447761194
2624089,24,23,0.5106382978723404
1173729,23,31,0.42592592592592593
879977,16,23,0.41025641025641024
705773,12,16,0.42857142857142855
108827,11,9,0.55
374693,10,11,0.47619047619047616
